templates/zerver/login.html

```diff
@@ -10,7 +10,12 @@
 <div class="app login-page split-view new-style flex full-page">
     <div class="inline-block">
         <div class="lead">
-            <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
+            <h1 class="get-started">
+                {{ _("Log in") }}
+                {% if email_change %} {{ _("to confirm email change") }} {% else %}
+                {{ _("to Zulip") }}
+                {% endif %}
```

Comment:
This is giving the raw html content to be like this containing spaces and `\n`'s
```
Log in\n                \n                to Zulip\n  
```
which inturn is causing a test(`login_login_preview` in `test_auth_backends.py`) fail that asserts that `Login in to Zulip` is present.

(It looks normal in the rendered HTML. Here's a screenshot to help us decide we actually want it there
![image](https://user-images.githubusercontent.com/44665669/87704704-69237f80-c7ba-11ea-8ff9-3663de4837ef.png)

Reply:
I'm confused if we should assert that `Log in\n                \n                to Zulip\n  ` is present or check if `Log in` and `to Zulip` are present or think of a different place to have that text in?

Reply:
Changed it to have two complete sentences rather than breaking the sentence just to save two words.